wizeline/wize-q-front
# WIP-226 - changed permissions of start.sh script to 755

#### What does this PR do?
Changes the permissions of the start.sh file from 644 to 755
#### How should this be manually tested?
1. build the image `docker build -t wizeq-front .`
2. Run the container: `docker run --rm -ti  -p 3000:3000 --env-file /path/to/your/project/.env --name wize-front-instance wizeq-front`
3. It should be up and running! Try entering [http://localhost:3000](http://localhost:3000) in your browser.  The login screen should be displayed
#### Any background context you want to provide?
CI failed because the permissions of the start.sh file were not set correctly
#### What are the relevant tickets?
https://wizeline.atlassian.net/browse/WIP-226
